althea_kernel_interface/src/exit_client_tunnel.rs

```diff
@@ -176,6 +195,44 @@ impl dyn KernelInterface {
         Ok(())
     }
 
+    pub fn set_ipv6_route_to_tunnel(&self, exit_ip: IpAddr) -> Result<(), Error> {
+        let ip_string = exit_ip.to_string();
+
+        // Find the gateway by look at existing babel routes
+        let output = self.run_command("ip", &["-6", "route"])?;
+        let stdout = String::from_utf8(output.stdout)?;
+        if !stdout.contains(&ip_string) {
+            error!("IPV6 ERROR: Unable to setup ipv6 default route, no babel route found to exit");
+            return Err(Error::RuntimeError(
+                "No babel route found to exit".to_string(),
+            ));
+        }
+        let split: Vec<&str> = stdout.split(&ip_string).collect();
+        let split: Vec<&str> = split[1].split(' ').collect();
+        let gw = split[2];
```

Comment:
As a method of getting the gateway I'm not quite sure this works, or if it does it's only by accident. There's no reason the exit ip and exit fe80 address on wg_exit have to be the same or similar at all. We should have the exit either tell us this address or have a more reliable way of getting it (not via babel, since there is no babel routing within the wg_exit tunnel)

Reply:
the function get_reply_ip in linklocal tools is probably a more solid way of doing this.